CONTRIBUTING.md

```diff
@@ -1 +1,109 @@
+- You can also select a location to clone to here.
+
+with Github desktop: `Clone a repository` -> select `URL`, paste `https://github.com/GITHUB_USERNAME/chunky`
+
+with CLI: `git clone git@github.com:GITHUB_USERNAME/chunky.git` or (outdated) `git clone https://github.com/GITHUB_USERNAME/chunky`
+
+##### Open the Project
+From inside IntelliJ (if you cloned with IntelliJ you can likely skip this step)
+- `File` -> `Open` -> Select the first `build.gradle` file within the cloned folder
+
+You may have to wait some time while IntelliJ indexes the project, and imports it. (The little loading bar in the bottom right) 
+
+##### Select the correct JDK version to use to build the project
+Java 17 SDK (e.g. openjdk-17-jdk on Ubuntu):
+- `File` -> `Project Structure` -> `Project` (should be the default)
+  - If IntelliJ detected your install: select the `SDK` dropdown, select a Java 17 JDK install
+  - If not: click `Edit`, then the little `+` at the top, `Add JDK` and find your Java 17 JDK install location (select the main jdk folder).
+    - Go back to the `Project` tab, and select the newly added jdk.
+
+TODO: ADD INFO ABOUT NULLABLE AND NOTNULL ANNOTATIONS (adding them to compiler options)
+
+##### Create a new branch for your change
+In the bottom right of IntelliJ, there is a fork button, by default it will say `master`, select it, at the top select `New Branch`, and name it after what you're intending to fix. (ideally something very short, no more than a few words). eg: `fix_scene_tab_typo` or `fix-scene-tab-typo`  is fine.
+ - Make sure the `Checkout branch` box is checked when creating a new branch (to actually change to it once it's created)
+
+##### Make your change
+Some useful IntelliJ keybinds to help you navigate around:
+ - `Ctrl-n`, Find a file (class) by name
+
+##### Commit your changes
+After making your changes, select the `Commit` button (looks like a green tick) in the top right
+Select all of the changes you want to make, and add a description of what you changed.
+
+### Testing
+Prior to making a Pull Request please test your changes. Within IntelliJ you can run chunky\src\java\se.llbit\chunky\main\Chunky.java:line64 assuming you have got IntelliJ setup correctly. Chunky's gradlew is 7.3.3.
+
+### Pull Request
+
+When you're finished with the changes, create a pull request, also known as a PR.
+- Provide a short but descriptive name and for more complex changes an extended description.
+- Don't forget to [link PR to issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) if you are solving one.
+- Enable the checkbox to [allow maintainer edits](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) so the branch can be updated for a merge.
+Once you submit your PR, a Docs team member will review your proposal. We may ask questions or request for additional information.
```

Comment:
A PR to the docs repoisotry?

Reply:
The links direct people to GitHub documentation on how to link issues and PRs, in addition to certain checkboxes that are recommended.

I fixed line 93 to remove the `Docs` confusion.